src/D2L.CodeStyle.UnsafeStaticCounter/Program.cs

```diff
@@ -49,33 +51,46 @@ internal sealed class Program {
 						enumerator.MoveNext();
 						outputFile = enumerator.Current;
 						break;
+					case "-b":
+						enumerator.MoveNext();
+						binDir = enumerator.Current;
+						break;
 					default:
 						throw new InvalidOperationException( $"unknown option: {enumerator.Current}" );
 				}
 			}
 
 			if( string.IsNullOrWhiteSpace( path ) ) {
-				throw new InvalidOperationException( "usage: UnsafeStaticsCounter.exe -d {rootDir} [-n {concurrency} -o {outputFile}]" );
+				throw new InvalidOperationException( "usage: UnsafeStaticsCounter.exe -d {rootDir} -b {binDir} [-n {concurrency} -o {outputFile}]" );
+			}
+			path = Path.GetFullPath( path );

```

Comment:
this fixes a related bug where relative checkout paths don't form properly, so we get no results (no csprojs).